autoload/airline/extensions/tagbar.vim

```diff
@@ -43,7 +44,7 @@ function! airline#extensions#tagbar#currenttag()
     endif
     " function tagbar#currenttag does not exist, if filetype is not enabled
     if s:airline_tagbar_last_lookup_time != localtime() && exists("*tagbar#currenttag")
-      let s:airline_tagbar_last_lookup_val = tagbar#currenttag('%s', '', s:flags)
+      let s:airline_tagbar_last_lookup_val = tagbar#currenttag('%s', '', s:flags, s:searchmethod)
```

Comment:
> let s:airline_tagbar_last_lookup_val = tagbar#currenttag('%s', '', s:flags, s:searchmethod)

Please make this: 

```vim
     let s:airline_tagbar_last_lookup_val = tagbar#currenttag('%s', '', s:flags, get(g:, 'airline#extensions#tagbar#searchmethod', '')
```

Reply:
Also, please add a section about the tagbar searchmethod setting `airline#extensions#tagbar#searchmethod` below: https://github.com/vim-airline/vim-airline/blob/master/doc/airline.txt#L1368-L1379 (and mention what values are valid for that). Thanks.

Reply:
I added documentation.

For the other comment - should `let s:searchmethod = get(g:, 'airline#extensions#tagbar#searchmethod', '')` on line 12 be removed then?

Reply:
Yes, please drop that line

Reply:
Done.